src/qt/askpassphrasedialog.cpp

```diff
@@ -272,9 +270,24 @@ bool AskPassphraseDialog::event(QEvent* event)
         // Detect Caps Lock key press.
         if (ke->key() == Qt::Key_CapsLock) {
             fCapsLock = !fCapsLock;
-            ui->capsLabel->setVisible(fCapsLock);
-            fCapsLock ? ui->capsLabel->setText(tr("WARNING: The Caps Lock key is on!")) : ui->capsLabel->clear();
         }
+
+        // Merge warning labels together if there's two warnings
+        bool validPassphrases = false;
+        validPassphrases = ui->passEdit2->text() == ui->passEdit3->text();
+        QString warningStr = "";
+        if (fCapsLock) warningStr += tr("WARNING: The Caps Lock key is on!");
+        if (fCapsLock && !validPassphrases) warningStr += "<br>";
+        if (!validPassphrases) warningStr += tr("WARNING: Passphrases do not match!");
```

Comment:
To not duplicate the "Warning" word twice, thinking that we could try the following organization:
```
Warning:
* The caps lock key is on!
* Passphrases do not match!
```

Reply:
Good idea, looks much nicer on the eyes this way; done.